src/components/Accordion.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { forwardRef, Ref } from 'react';
```

Comment:
Minor thing but could be just
```suggestion
import { forwardRef, Ref } from 'react';
```